src/testFixtures/kotlin/io/github/anki/testing/IntegrationTest.kt

```diff
@@ -0,0 +1,13 @@
+package io.github.anki.testing
+
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.testcontainers.junit.jupiter.Testcontainers
+
+
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+@AutoConfigureMockMvc
+@Testcontainers
+@SpringBootTest
+annotation class IntegrationTest
```

Comment:
The mock mvc annotation creates mock mvc that used at integration tests, but actually u won't add this annotation to repository test.

I suggest to add additional annotation

```suggestion
@Target(AnnotationTarget.CLASS)
@Retention(AnnotationRetention.RUNTIME)
@Testcontainers
@SpringBootTest
@Tag("integration")
annotation class IntegrationTest

@IntegrationTest
@AutoConfigureMockMvc
annotation class MVCTest

```